app/scripts/helpers/taginput.helper.js

```diff
@@ -1050,69 +1069,78 @@
                 },
                 getTextAutosizeThreshold: function() {
                     return autosizeThreshold;
-                }
+                },
             };
         }];
     });
 
 
-    /***
+    /** *
      * @ngdoc service
      * @name tiUtil
      * @module ngTagsInput
      *
      * @description
      * Helper methods used internally by the directive. Should not be called directly from user code.
      */
-    tagsInput.factory('tiUtil', ["$timeout", "$q", function($timeout, $q) {
-        var self = {};
+    tagsInput.factory('tiUtil', ['$timeout', '$q', function($timeout, $q) {
+        var utils = {};
 
-        self.debounce = function(fn, delay) {
+        utils.debounce = function(fn, delay) {
             var timeoutId;
+
             return function() {
                 var args = arguments;
+
                 $timeout.cancel(timeoutId);
-                timeoutId = $timeout(function() { fn.apply(null, args); }, delay);
+                timeoutId = $timeout(() => {
+                    fn(...args);
```

Comment:
It's might not matter in this case, but a correct translation would be `fn.call(null, ...args)`. When you do `fn(...args)` you are not binding `this` and the function might receive `window` or `undefined` as  this when it used to get `null` previously.

Reply:
this part was done by `eslint --fix` I hope they know what they are doing :wink: